### ISSUE TITLE
I lost all my information, suddenly

### ISSUE THREAD

**Issue by Altuijri** [screenisland/kanban-kit#40]
Hi
I really love the app, the simplicity makes me use it instead of Kanban board. Tags are wonderful idea. 
I downloaded the app yesterday and used it to track my plans too, I spent 7 hours yesterday moving my information from To-Do apps to the Kanban app. I saved the information, I closed the app, reopened it many times. When i put the app on the full-screen mode, the app freezes. I turned it off and reopened it again, but I lost every single piece of information and I returned to the point zero.
I am afraid that such thing can happen again, and wonder how to protect my information? Any way to sync it with cloud?

**Comment by gobijan** [comment]
I am very sorry to hear that. This is weird. And has never been the case for me but I heard it from a view users that in rare cases (after a crash) data is gone. Best way for now is to Export the tasks as json from time to time and to keep the export in the cloud. 
Also: Do you have a crash report or something?

**Comment by Altuijri** [comment]
No, unfortunately I don’t have, but if whenever the app crashes, the information is gone, despite saving it, that is a problem, a serious one. I am including a very sensitive data and the input process requires lots of time.

Ahmad

**Comment by CyberonAustralia** [comment]
I have also had exactly the same problem! it is a pretty basic thing to ensure that files are written out (and can be read back) - so much for my weeks worth of work!!!

**Comment by mattwallington** [comment]
This just happened to me as well and it's very frustrating.  My app didn't crash, I closed it normally yet when I opened it back up, everything was gone.  I spent a day getting everything into there.  You need to fix this or not charge so much for the app.

**Comment by mattfuller** [comment]
Hello - I had the same experience as Matt Wallington.  I closed the app normally and lost all the information I set up.  It's frustrating to have spent time putting information in to set up my work for the next day and then lose it first thing in the morning.

**Comment by gobijan** [comment]
Guys this seems to be a bug with apples implementation for webviews. It seems there is not much I can do at the moment. Can you Tell me what lead to loosing data? Does it always happen? Please save your tasks into a file from time to time. This bug started to appear from time to time after some OS X updates were rolled out.

**Comment by mattfuller** [comment]
I usually try to keep the app open while I'm working, so it's dormant for a bit while I'm working in other apps. Then it may or may not lock up.  Is there a way to include a better way to save files, such as an autosave? I've had this app for a week and it's froze and lost data at least 4 times.  Frustrating.

**Comment by uboweja** [comment]
Hi, I purchased the app 5 days back and was quite excited to use it, loved it for simplicity. Faced two issues, a minor and a major one. In a minor issue, the due date calculation for one of the tasks automatically got updated to Due after 2 days when it was due today. I managed to sort that by changing it twice. The second and the major issue is that I restarted the app and lost all the tasks! Very disappointing as I was just about to start enjoying the app as a refreshing change from busy apps.
